src/NuGet.Core/NuGet.DependencyResolver.Core/ResolverUtility.cs

```diff
@@ -24,14 +24,15 @@ public static class ResolverUtility
             NuGetFramework framework,
             string runtimeIdentifier,
             RemoteWalkContext context,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken,
+            LibraryDependency[] projectDependencies = null)
```

Comment:
I'm really not a fan of optional types in public APIs. I prefer a new overload, which would also prevent a breaking ABI change. A new overload would also keep the convention of keeping the CancellationToken as the last parameter.